config/default_config.yml

```diff
@@ -72,3 +72,6 @@ direction:
 category:
   enabled: false
   maxPlaces: 40
+
+cors:
+  enabled: true
```

Comment:
This name does not describe the exact feature. It's actually not directly related to CORS.  
What about `telemetry.sendQueryContextHeaders` ?